scene/debugger/scene_debugger.cpp

```diff
@@ -1641,7 +1641,11 @@ void RuntimeNodeSelect::_physics_frame() {
 				// Allow forcing box selection when an item was clicked.
 				selection_drag_state = SELECTION_DRAG_MOVE;
 			} else if (items.is_empty()) {
+#ifndef _3D_DISABLED
 				if (!selected_ci_nodes.is_empty() || !selected_3d_nodes.is_empty()) {
+#else
+				if (!selected_ci_nodes.is_empty()) {
+#endif // _3D_DISABLED
```

Comment:
```suggestion
#ifdef _3D_DISABLED
				if (!selected_ci_nodes.is_empty()) {
#else
				if (!selected_ci_nodes.is_empty() || !selected_3d_nodes.is_empty()) {
#endif // _3D_DISABLED
```
For all these, general preference not not have the first option be negated